### PR TITLE
Most ratvarian text is now passed through text2ratvar()

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Revenant scriptures now announce to all servants and ghosts when used instead of having a visible message.
/:cl:

Accidentally fixes some countdown code to be more simple(coiax what the fuck why was that using an overlay)
Retcons Inath-neq's name to have a lower-case N, so that the regex for it doesn't need to capitalize it.
Redoes the tiny bit of Nzcrentr lore.
I might have fixed a bug or two? It's kind of hazy.

### CODE REVIEW

**Comment by Man-Shroom-** [comment]
YEAH LANGUAGE SYSTEMS WOOOOOO

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -335,9 +335,6 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 			if(L.reagents && L.reagents.has_reagent("holywater"))
 				L.reagents.remove_reagent("holywater", 1000)
 				L << "<span class='heavy_brass'>Ratvar's light flares, banishing the darkness. Your devotion remains intact!</span>"
-	for(var/mob/living/silicon/ai/A in range(1, get_turf(invoker))) //Seems necessary because AIs don't count as hearers for some reason
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: hearers is a BYOND proc, augh. this comment was just wrong.

File: code/game/gamemodes/clock_cult/clock_unsorted.dm [hunk 2]
```diff
@@ -239,16 +239,18 @@ List of nuances:
...
 
 //Regexes used to alter english to ratvarian style
 #define RATVAR_OF_MATCH				regex("(\\w)\\s(\[oO]\[fF])","g")
 #define RATVAR_OF_REPLACEMENT 		"$1-$2"
+#define RATVAR_GUA_MATCH			regex("(\[gG]\[uU])(\[aA])","g")
+#define RATVAR_GUA_REPLACEMENT		"$1-$2"
 #define RATVAR_TH_MATCH				regex("(\[tT]\[hH]\\w)(\\w)","g")
 #define RATVAR_TH_REPLACEMENT		"$1`$2"
-#define RATVAR_TI_MATCH				regex("(\[tT]\[iI])(\\w{2})","g")
+#define RATVAR_TI_MATCH				regex("(\[tT]\[iI])(\\w)","g")
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_unsorted.dm]: I swear the ruling for TI said for 2 letters, oh well

File: code/game/gamemodes/clock_cult/clock_unsorted.dm [hunk 3]
```diff
@@ -239,16 +239,18 @@ List of nuances:
...
 - Although "Ratvar" translates to "Engine" in English, the word "Ratvar" is used regardless of language as it is a proper noun.
- - The same rule applies to Ratvar's four generals: Nezbere (Armorer), Sevtug (Fright), Nzcrentr (Amperage), and Inath-Neq (Vangu-Ard), although these words can be used in proper context if one is
+ - The same rule applies to Ratvar's four generals: Nezbere (Armorer), Sevtug (Fright), Nzcrentr (Amperage), and Inath-neq (Vangu-Ard), although these words can be used in proper context if one is
    not referring to the four generals and simply using the words themselves.
 */
 
 //Regexes used to alter english to ratvarian style
 #define RATVAR_OF_MATCH				regex("(\\w)\\s(\[oO]\[fF])","g")
 #define RATVAR_OF_REPLACEMENT 		"$1-$2"
+#define RATVAR_GUA_MATCH			regex("(\[gG]\[uU])(\[aA])","g")
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_unsorted.dm]: please write the english form of this rule
(gua -> gu-a)

**Pull request merged**